docs/content/latest/manage/backup-restore/point-in-time-restore.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Point-in-Time Restore for YSQL
+headerTitle: Point-in-time restore
+linkTitle: Point-in-time restore
+description: Restore data from a specific point in time in YugabyteDB for YSQL
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+aliases:
+menu:
+  latest:
+    identifier: point-in-time-restore
+    parent: backup-restore
+    weight: 704
+isTocNested: true
+showAsideToc: true
+---
+
+The point-in-time restore feature allows you to restore the state of your cluster's data from a specific point in time. This can be relative, such as "three hours ago", or an absolute timestamp.
+
+_Point-in-time restores_ (also referred to here as PITR) and _incremental backups_ go hand in hand. These two features help in recovering from a number of error or failure scenarios by allowing the database to be restored to a specific point in time (in the past).
+
+Point-in-time restores and incremental backups depend on _full backups_ (also referred to as base backups). A full backup, as the name suggests, is a complete transactional backup of data up to a certain point in time. The entire data set in the database is backed up for all of the namespaces and tables you selected. Full backups are resource-intensive, and can consume considerable amounts of CPU time, bandwidth, and disk space.
+
+To learn more about YugabyteDB's point-in-time restore feature, refer to the [Recovery scenarios](#recovery-scenarios), [Features](#features), [Use cases](#use-cases), and [Limitations](#limitations) sections on this page. For more details on the `yb-admin` commands, refer to the [Backup and snapshot commands](../../../admin/yb-admin#backup-and-snapshot-commands) section of the yb-admin documentation.
+
+## Try out the PITR feature
+
+There are several recovery scenarios [for YSQL](../../../explore/backup-restore/point-in-time-restore-ysql/) and [for YCQL](../../../explore/backup-restore/point-in-time-restore-ycql/) in the Explore section.
+
+## Recovery scenarios
+
+### App and operator errors
```

Comment:
since GH doesn't make it very obvious, I'm assuming most of the below is just moved over from the old version, right?

Reply:
Yep